src/Microsoft.DotNet.Interactive.ExtensionLab.Tests/SqlKernelsExtensionTests.cs

```diff
@@ -90,6 +90,17 @@ private static IEnumerable<IEnumerable<IEnumerable<(string name, object value)>>
             } while (reader.NextResult());
         }
 
+        [Fact]
+        public void can_alias_duplicate_column_names_in_sql_results()
+        {
+            var testNames = new string[] { "apple", "banana", "apple", "orange", "orange", "BANANA", "orange", "BaNaNa" };
+            var expectedAliases = new string[] { "apple", "banana", "apple (1)", "orange", "orange (1)", "BANANA", "orange (2)", "BaNaNa" };
+
+            SqlKernelUtils.AliasDuplicateColumnNames(testNames);
+
+            Assert.Equal(expectedAliases, testNames);
```

Comment:
Please use FluentAssertions, e.g. `expectedAliases.Should().BeEquivalentTo(testNames);`